application/templates/cms/_measures_with_version_history.html

```diff
@@ -8,7 +8,7 @@
     <thead>
     <tr>
         <th>
-            Measure name
+            Page name
```

Comment:
"Page title" might better match the "Title" field on the edit page.

Reply:
Yep, cool, please change to 'Page title'

Reply:
👍 done.